test/unit/token-manager-helper.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
The purpose of this might be more clear if it was in a `utils` file, or something, within the `test/unit` directory. It's perfectly fine here, too - just a thought.

Reply:
fixed